packages/ui/lib/components/Form/AddressInput.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  InputHTMLAttributes,
+  ForwardedRef,
+  ReactNode,
+  useState,
+} from 'react'
+import type { Size, SizeStyleProp } from '../../types'
+import { forwardRef } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { FieldLayout } from './FieldLayout'
+import { Icon } from '../Icon/Icon'
+import { FaWallet as WalletIcon } from 'react-icons/fa'
+import { IconBaseProps } from 'react-icons'
+import { minifyAddress } from '../../utils'
+
+export interface Props
+  extends Omit<
+    InputHTMLAttributes<HTMLInputElement>,
+    'size' | 'id' | 'children'
+  > {
+  label?: string
+  size?: Size
+  description?: ReactNode
+  withIcon?: boolean
+  isTruncated?: boolean
+  web3Service: any
+  localForm: any;
```

Comment:
```suggestion
  localForm: UseFormReturn;
```

@SouravInsights add typing here and import it from `import { UseFormReturn } from 'react-hook-form'`